content/en/docs/started/installing-kubeflow.md

```diff
@@ -153,6 +153,20 @@ The following table lists distributions which are <em>maintained</em> by their r
           <a href="https://nutanix.github.io/kubeflow-manifests">Website</a>
         </td>
       </tr>
+      <tr>
+        <td>
+          QBO
+        </td>
+        <td>
+          {{% qbo/latest-version %}} <sup>[<a href="https://github.com/alexeadem/qbo-ce/blob/main/CHANGELOG.md">Release Notes</a>]</sup>
+        </td>
+        <td>
+          QBO Kubernetes Engine (QKE) Community and Cloud Editions
```

Comment:
This is unreasonably long, how about we use this:

```suggestion
          QBO Kubernetes Engine
```

Reply:
Van we do this `QBO Kubernetes Engine (QKE)` instead?